src/PhpLibre.php

```diff
@@ -0,0 +1,227 @@
+<?php
+
+namespace CidiLabs\PhpLibre;
+
+use DOMDocument;
+use DOMXPath;
+use Ramsey\Uuid\Uuid;
+
+class PhpLibre {
+
+    private $bin;
+
+
+    public function __construct($bin = 'soffice')
+    {
+        $this->bin = $bin;
+    }
+
+    public function convertFile($fileUrl, $fileData, $format) {
```

Comment:
I updated aws-polly library to accept a single param `$options`. I think it provides more flexibility. 

https://github.com/cidilabs/aws-polly/blob/fc42cb61a29e1dd4d2de519c09171f95a956e624/src/AwsPollyFileConversionProvider.php#L39

Reply:
Okay I'll update to fit that scheme.